snowpack/src/build/optimize.ts

```diff
@@ -42,14 +26,17 @@ interface ScannedHtmlEntrypoint {
   getLinks: (rel: 'stylesheet') => cheerio.Cheerio;
 }
 
+// The manifest type is the one from ESBuild, but we might delete the outputs key
+type SnowpackMetaManifest = Omit<esbuild.Metafile, 'outputs'> & Partial<esbuild.Metafile>;
```

Comment:
Is this type too weird for your taste? It worked with TS 4.1 and I have updated TS to 4.2.3 to check that it keeps working.

Reply:
that works